src/Components/Navbar/Navbar.tsx

```diff
@@ -26,9 +26,12 @@ export function Navbar(props: INavbar) {
   }
   const router = useRouter();
 
-  const handleLogout = async () => {
+  const handleLogout = () => {
     try {
```

Comment:
unnecessary trycatch

Reply:
I have discussed with rajesh sir I think it is important to use the try catch in this logout functionalty